packages/types/src/posts.ts

```diff
@@ -138,6 +138,8 @@ export type PostsState = {
 export declare type OpenGraphMetadataImage = {
     secure_url?: string;
     url: string;
+    type?: string;
+    format?: string;
```

Comment:
that comes from a different component which needed that ... I just cannot remember where that came from ... It is very likely an error on that side, but it never showed since that one was still a `jsx` as far as I can remember. I will investigate this later in more detail!

Reply:
Ok, so the property `format` came from a requirement in the `ExternalImage` component. I am not sure why I added that field to the `OpenGraphMetadataImage` type though, so I reverted it now. NICE CATCH!